packages/jellyfish-api-core/__tests__/category/spv/listAnchors.test.ts

```diff
@@ -0,0 +1,134 @@
+      expect(team.auth.includes(GenesisKeys[1].operator.address))
+      expect(team.auth.includes(GenesisKeys[2].operator.address))
+      expect(team.confirm.includes(GenesisKeys[0].operator.address))
+      expect(team.confirm.includes(GenesisKeys[1].operator.address))
+      expect(team.confirm.includes(GenesisKeys[2].operator.address))
+    }
+
+    await tGroup.anchor.generateAnchorAuths(2, initOffsetHour)
+
+    await tGroup.get(0).container.waitForAnchorAuths(tGroup.length())
+
+    // check each container should be quorum ready
+    for (let i = 0; i < tGroup.length(); i += 1) {
+      const { container } = tGroup.get(i % tGroup.length())
+      const auths = await container.call('spv_listanchorauths')
+      expect(auths.length).toStrictEqual(2)
+      expect(auths[0].signers).toStrictEqual(tGroup.length())
+    }
+
+    await createAnchor()
+    await tGroup.get(0).container.waitForBlockHeight(75)
+    await tGroup.waitForSync()
+
+    await createAnchor()
+    await tGroup.get(0).container.waitForBlockHeight(90)
+    await tGroup.waitForSync()
+
+    await createAnchor()
+    await tGroup.get(0).container.waitForBlockHeight(105)
+    await tGroup.waitForSync()
+
+    await createAnchor()
+    await tGroup.get(0).generate(1)
+    await tGroup.waitForSync()
+
+    await tGroup.get(0).container.call('spv_setlastheight', [6])
+  }
+
+  async function createAnchor (): Promise<void> {
+    const rewardAddress = await tGroup.get(0).rpc.spv.getNewAddress()
+    await tGroup.get(0).rpc.spv.createAnchor([{
+      txid: '11a276bb25585f6973a4dd68373cffff41dbcaddf12bbc1c2b489d1dc84564ee',
+      vout: 2,
+      amount: 15800,
+      privkey: 'b0528d87cfdb09f72c9d10b7b3cc00727062d93537a3e8abcf1fde821d08b59d'
+    }], rewardAddress)
+  }
+
+  it('should listAnchors', async () => {
+    const anchors = await tGroup.get(0).rpc.spv.listAnchors()
+    expect(anchors.length).toStrictEqual(4)
+    for (const anchor of anchors) {
+      expect(typeof anchor.btcBlockHeight).toStrictEqual('number')
+      expect(typeof anchor.btcBlockHash).toStrictEqual('string')
+      expect(typeof anchor.btcTxHash).toStrictEqual('string')
+      expect(typeof anchor.previousAnchor).toStrictEqual('string')
+      expect(typeof anchor.defiBlockHeight).toStrictEqual('number')
+      expect(typeof anchor.defiBlockHash).toStrictEqual('string')
+      expect(typeof anchor.rewardAddress).toStrictEqual('string')
+      expect(typeof anchor.confirmations).toStrictEqual('number')
+      expect(typeof anchor.signatures).toStrictEqual('number')
+      expect(typeof anchor.anchorCreationHeight).toStrictEqual('number')
+      expect(typeof anchor.active).toStrictEqual('boolean')
+    }
+  })
+
+  it('should listAnchors with minBtcHeight', async () => {
+    const anchors = await tGroup.get(0).rpc.spv.listAnchors({ minBtcHeight: 10 })
+    expect(anchors.length).toStrictEqual(0)
+  })
+
+  it('should listAnchors with maxBtcHeight', async () => {
```

Comment:
all anchors created in the setup() were at btcheight 0, so maybe we can spread anchor creation across different btc block heights and demonstrate that these `maxBtcHeight, minBtcHeight, minConfs,  maxConfs` params truly filter out the unnecessary?